src/styles/custom.css

```
@@ -27,14 +27,23 @@
   top: var(--sl-announcement-height) !important;
   height: calc(var(--sl-nav-height) - var(--sl-announcement-height)) !important;
 }
-:root[data-promo-dismissed] .header {
+:root[data-promo-dismissed] .page > header.header {
   top: 0 !important;
 }
 
+@media (max-width: 49.999rem) {
+  starlight-menu-button button[aria-controls='starlight__sidebar'] {
+    top: calc(
+      var(--sl-announcement-height) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)